analysis/gradle.properties

```diff
@@ -1,4 +1,5 @@
 # Project-wide Gradle settings.
+
```

Comment:
Delete empy line

Reply:
The ids that I have added are being used in the e2e tests to target the particular element.  That is true that the search and blacklist are very similar . The difference comes in the return values and what we want to achieve one returns an array of nodes and the other we need a boolean as we iterate over all the nodes in a map. In the other issue 901 I have tried to refactor this part some more.